Shared/Samples/Validate utility network topology/ValidateUtilityNetworkTopologyView.Model.swift

```diff
@@ -0,0 +1,448 @@
+        func setup() async throws {
+            // Add the credential to access the web map.
+            try await ArcGISEnvironment.authenticationManager.arcGISCredentialStore.add(.publicSample)
+            
+            try await setupMap()
+            try await setupUtilityNetwork()
+            try await setupTraceParameters()
+            
+            // Set the initial states using utility network's capabilities.
+            guard let utilityNetworkCapabilities = utilityNetwork.definition?.capabilities else { return }
+            canGetState = utilityNetworkCapabilities.supportsNetworkState
+            canTrace = utilityNetworkCapabilities.supportsTrace
+            canValidateNetworkTopology = utilityNetworkCapabilities.supportsValidateNetworkTopology
+            canClearSelection = false
+            
+            statusMessage = """
+            Utility Network Loaded
+            Tap on a feature to edit.
+            Tap 'Get State' to check if validating is necessary or if tracing is available.
+            Tap 'Trace' to run a trace.
+            """
+        }
+        
+        /// Sets up and loads the web map.
+        private func setupMap() async throws {
+            statusMessage = "Loading web map..."
+            
+            // Create a portal item using the portal and id for the Naperville Electric web map.
+            let portal = Portal(url: .sampleServerPortal, connection: .authenticated)
+            let portalItem = PortalItem(portal: portal, id: .napervilleElectric)
+            
+            // Set the portal item to the map and load the map.
+            map.item = portalItem
+            map.initialViewpoint = Viewpoint(center: Point(x: -9815160, y: 5128710), scale: 3640)
+            try await map.load()
+            
+            // Set the map to load in persistent session mode (workaround for server caching issue).
+            // https://support.esri.com/en-us/bug/asynchronous-validate-request-for-utility-network-servi-bug-000160443
+            map.loadSettings.featureServiceSessionType = .persistent
+            
+            // Add labels to the map to visualize attribute editing.
+            addLabels(to: .deviceTableName, for: .deviceStatusField, color: .blue)
+            addLabels(to: .lineTableName, for: .nominalVoltageField, color: .red)
+        }
+        
+        /// Loads the utility network and switches to a new version on the service.
+        private func setupUtilityNetwork() async throws {
+            statusMessage = "Loading utility network..."
+            
+            // Get the utility network from the map.
+            utilityNetwork = map.utilityNetworks.first
+            try await utilityNetwork.load()
+            
+            // Create service version parameters to restrict editing and tracing on a random branch.
+            let randomString = UUID().uuidString
```

Comment:
"Unique" is probably more appropriate than "random".

```suggestion
            let uniqueString = UUID().uuidString
```